CHANGES.md

```diff
@@ -1,3 +1,21 @@
+## edge-20.5.1
+
+* CLI
+  * Fixed all commands to use kubeconfig's default namespace if specified
+    (thanks @Matei207!)
+  * Added multicluster checks to the `linkerd check` command
+  * Hid development flags in the `linkerd install` command for release builds
+* Controller
+  * Added ability to configure Prometheus Alertmanager (more documentation and
+    changes to come for this!) (thanks naseemkullah!)
+* Web UI
```

Comment:
We could also add #4319 here! But not any user facing changes unless users manually remove the `garfana-addr` field in `linkerd-web` manifest

Reply:
@Pothulapati I left this out for now since it is such a specific change that needs to occur for it to be considered user facing. Once all the PRs related to making Grafana a soft requirement have merged, it will be easier to make this a point in the release notes.